fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -82,7 +83,7 @@ public void resetColumns(SegmentMetadata segmentMetadata, DimensionDictionary di
         setAvailability(new ImmutableAvailability(schema, dimensionIntervals, metricIntervals, dimensionDictionary));
     }
 
-    public void setAvailability(Availability availability) {
+    public void setAvailability(Map<Column, List<Interval>> availability) {
```

Comment:
Does it make sense to add by giving a map instead of have to construct a Availability from map first?